messages/deploy-remote/messages.go

```diff
@@ -42,12 +42,10 @@ var (
 	AskInputName                         = "Type the new name:"
 	ProjectNameMessage                   = "Using the same name as your project to create the bucket\n"
 	AskCreateCacheSettings               = `Azion CLI offers to create the following Cache Settings specifications:
-  - Browser Cache Settings: Override Cache Settings
-  - Maximum TTL for Browser Cache Settings (in seconds): 7200
-  - Edge Application Cache Settings: Override Cache Settings
-  - Maximum TTL for Edge Application Cache Settings (in seconds): 7200
+  - Browser Cache Settings: Override Cache Settings - Maximum TTL in seconds: 7200
```

Comment:
```suggestion
  - Browser Cache Settings: Override Cache Settings
```

Do we need this part ` - Maximum TTL in seconds: 7200`? 🤔

Reply:
Nice question. I chose to leave the maximum TTL because I understood this is the value of the specific configurations offered by the CLI but it could be different. Maybe @PatrickMenoti might clarify this point